theia-extensions/launcher/src/browser/create-launcher-contribution.ts

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (C) 2022 EclipseSource and others.
+ * Copyright (C) 2022-2024 EclipseSource and others.
```

Comment:
I'm not sure we update the headers for files we change, usually. Not sure if there is a policy and what it is.